docs/en/configuration/report-log.md

```diff
@@ -0,0 +1,52 @@
+# Report Log
+
+## Overview
+
+In `skywalking-php`, the log level configuration is managed using the `skywalking_agent.psr_logging_level` directive in your `php.ini` file. This configuration defines the minimum log level that will be reported to SkyWalking. The log levels are based on PSR-3 standards and allow you to control the verbosity of the logs sent to SkyWalking.
+
+## Configuration
+
+You can set the `skywalking_agent.psr_logging_level` in your `php.ini` file:
+
+```ini
+skywalking_agent.psr_logging_level = Info
+```
+
+The possible values for this configuration are:
+
+- `Off`: No logs will be reported to SkyWalking.
+- `Debug`: Logs with the level "Debug" or higher will be reported.
+- `Info`: Logs with the level "Info" or higher will be reported.
+- `Notice`: Logs with the level "Notice" or higher will be reported.
+- `Warning`: Logs with the level "Warning" or higher will be reported.
+- `Error`: Logs with the level "Error" or higher will be reported.
+- `Critical`: Logs with the level "Critical" or higher will be reported.
+- `Alert`: Logs with the level "Alert" or higher will be reported.
+- `Emergency`: Logs with the level "Emergency" or higher will be reported.
+
+### Default Value
+
+The default value for `skywalking_agent.psr_logging_level` is set to `Off`, which means no logs will be reported to SkyWalking unless specified otherwise.
```

Comment:
```suggestion
The default value for `skywalking_agent.psr_logging_level` is set to `Off`, which means no log will be reported to SkyWalking unless specified otherwise.
```